src/cpp/common/channel_filter.cc

```diff
@@ -71,7 +71,7 @@ void CallData::SetPollsetOrPollsetSet(grpc_call_element* elem,
 namespace internal {
 
 void RegisterChannelFilter(
-    grpc_channel_stack_type stack_type, int priority,
+    grpc_channel_stack_type stack_type, int,
```

Comment:
How about just removing the priority arg completely and updating the callers?

Reply:
I'm going to delete this file entirely soon.